react-attempts/src/components/PlusButton/PlusButton.jsx

```diff
@@ -0,0 +1,8 @@
+ const PlusButton = (props) => {
```

Comment:
```suggestion
 const PlusButton = ({ increment }: TypeForProps) => {
```
Пропсам надо указывать тип и деструктурировать их либо прям тут, либо в самом компоненте. Так сразу видно что именно пришло.